catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/SolrProviderRealTimeQueryTest.java

```diff
@@ -60,6 +63,8 @@
 
   public static final String MASKED_ID = "scp";
 
+  private static final float DEFAULT_RELEVANCE = 100f;
```

Comment:
✏️ As it appears you're always comparing `DEFAULT_RELEVANCE` as a double, it would be cleaner to define it here as a double. `100d` would avoid the automatic upcast from float to double, but either way if you just declare this as a double you can remove the casts in your tests.